Lib/idlelib/pyshell.py

```diff
@@ -1247,6 +1247,18 @@ def restart_shell(self, event=None):
         self.interp.restart_subprocess(with_cwd=True)
 
     def showprompt(self):
+        """Display the prompt in the shell window.
```

Comment:
```suggestion
        "Display the prompt."
```
and delete rest.  I will soon propose changing handing of prompt.